collect_stata/write_json.py

```diff
@@ -185,9 +185,10 @@ def generate_statistics(
     Output:
     stat: OrderedDict
     """
-
+    counter = 0
     for elem in metadata:
-        logging.info("%s", str(len(metadata)))
+        counter = counter + 1
+        logging.info("%s/%s", str(counter), str(len(metadata)))
```

Comment:
Is this needed?
I do not see the usefulness of the logged information.

Also:

* str(len(metadata)) does not change, set it once outside the loop if you want to use it.
* You do not need an extra counter variable.

  + enumerate(metadata) returns value and index position as a tuple, use that.